src/styles/_franklin-settings.scss

```diff
@@ -0,0 +1,102 @@
+// Font families
+$fs-base-font-face: 'Lato', sans-serif;
+$fs-headers-font-face: 'Source Sans Pro', sans-serif;
```

Comment:
This might be due to something else, but it seems there is an issue with antialiasing of the title font. Tested on chrome but might be other browsers too. Have you added this somewhere:
* {
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

Reply:
Added the anti-aliasing now!